### PR TITLE
fix: fix build on macos, closes #580

### LINKED ISSUE #580: Wry 0.16.1 does not build on macos
**Describe the bug**
The build of the crate fails on macos.

This is the error message:
```
error[E0308]: mismatched types
   --> /Users/me/.cargo/registry/src/github.com-1ecc6299db9ec823/wry-0.16.1/src/webview/wkwebview/mod.rs:420:12
    |
420 |         if can_set_titlebar_style {
    |            ^^^^^^^^^^^^^^^^^^^^^^ expected `bool`, found `i8`

error[E0308]: mismatched types
   --> /Users/me/.cargo/registry/src/github.com-1ecc6299db9ec823/wry-0.16.1/src/webview/wkwebview/mod.rs:549:10
    |
549 |       if can_print {
    |          ^^^^^^^^^ expected `bool`, found `i8`

For more information about this error, try `rustc --explain E0308`.
error: could not compile `wry` due to 2 previous errors
warning: build failed, waiting for other jobs to finish...
error: build failed
```

**Steps To Reproduce**
Try to build the crate using `cargo build`.

**Expected behavior**
The build should succeed.

**Platform and Versions (please complete the following information):**
OS: MacOS 12.3.1
Rustc: rustc 1.60.0 (7737e0b5c 2022-04-04)

**Would you assign yourself to resolve this bug?**
- [x] Yes
- [ ] No

**Additional context**
The bug seems to have been introduced in eb2dddb611f7fadf35bf7d7c32cb6d054da9fe9e.

However I have no idea how it was possible, that this slipped the Github Actions CI?

### PR DESCRIPTION
**Pull request by FrankenApps**
### What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Docs
- [ ] New Binding issue #___
- [ ] Code style update
- [ ] Refactor
- [x] Build-related changes
- [ ] Other, please describe:

### Does this PR introduce a breaking change?

- [ ] Yes, and the changes were approved in issue #___
- [x] No

### Checklist
- [x] When resolving issues, they are referenced in the PR's title (e.g `fix: remove a typo, closes #___, #___`)
- [x] A change file is added if any packages will require a version bump due to this PR per [the instructions in the readme](https://github.com/tauri-apps/tauri/blob/dev/.changes/readme.md).
- [ ] I have added a convincing reason for adding this feature, if necessary

### Other information
The problem was initially described in #580.

### CODE REVIEW

**Review by amrbashir — DISMISSED** [review]

**Comment by amrbashir** [comment]
Could you add a change file in .`changes` dir? Thanks for the report and the fix, I will merge and publish ASAP.

**Comment by FrankenApps** [comment]
I added the markdown file in the `.changes` directory. Hopefully it is correct. I found the explanation in the readme a bit cumbersome. Maybe it would be helpful for future contributors to point out, that in this repo there is only one package (namely `wry`)?

File: src/webview/wkwebview/mod.rs [hunk 1]
```diff
@@ -417,7 +417,7 @@ impl InnerWebView {
           ns_window,
           respondsToSelector: sel!(setTitlebarSeparatorStyle:)
         ];
-        if can_set_titlebar_style {
+        if can_set_titlebar_style != NO {
```
Comment (amrbashir) [inline src/webview/wkwebview/mod.rs]:
```suggestion
        if can_set_titlebar_style == YES {
```

File: src/webview/wkwebview/mod.rs [hunk 2]
```diff
@@ -546,7 +546,7 @@ r#"Object.defineProperty(window, 'ipc', {
         self.webview,
         respondsToSelector: sel!(printOperationWithPrintInfo:)
       ];
-      if can_print {
+      if can_print != NO {
```
Comment (amrbashir) [inline src/webview/wkwebview/mod.rs]:
```suggestion
      if can_print == YES {
```

**Review by amrbashir — CHANGES_REQUESTED** [review]

**Review by amrbashir — APPROVED** [review]

**Comment by amrbashir** [comment]
Thanks, I will merge once CI passes

**Pull request merged**